### DISCUSSION TITLE
posts/javascript-closure-deep-dive-application

### DISCUSSION THREAD

**Discussion by giscus** [witch-factory/witch-next-blog#47] [General]
# posts/javascript-closure-deep-dive-application

클로저, 대체 무엇이며 어디에 쓰는 거지?

https://witch.work/posts/javascript-closure-deep-dive-application

**Comment by owni14** [comment]
복잡한 개념들을 이해하기 쉽게 설명해주셔서 도움이 많이 된거 같아요!

**Comment by hi6724** [comment]
홀리몰리
